package.json

```diff
@@ -4,21 +4,35 @@
   "description": "Plugin to make TypeScript's JSDoc type annotations work with JSDoc",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint index.js",
+    "pretest": "npm run lint",
+    "test": "node test/test.js"
   },
   "keywords": [
     "jsdoc",
     "typescript"
   ],
+  "files": [
+    "index.js"
+  ],
   "license": "BSD-2-Clause",
-  "peerDependencies": {
-    "jsdoc": ">=3.6.0"
-  },
```

Comment:
Why was jsdoc removed from peer dependencies? Isn't the package invalid now as it still `require`s it?

Reply:
This project does not depend on `jsdoc`. It uses `jsdoc` in the tests, but loads them through `npx`.